recipes/bindsnet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "bindsnet" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Latest version on pypi is 0.2.7 and comes from https://github.com/BINDS-LAB-UMASS/bindsnet
Is that repo abandoned and https://github.com/BindsNET/bindsnet a fork that took officially over?

Reply:
Disclaimer first: I am not involved in the development of BindsNet - just a user.

As https://github.com/BINDS-LAB-UMASS/bindsnet hasn't been updated for years and has a mere 35 stars, I'd say it's abandoned.

This compares to https://github.com/BindsNET/bindsnet which indeed builds on top of the last commit in https://github.com/BINDS-LAB-UMASS/bindsnet. It has very recent and frequent commits, and >1000 stars. So https://github.com/BindsNET/bindsnet seems to be the right repo to use.

Reply:
OK, I'm not sure of the conda-forge policy in this case. You could maybe try to open an issue in https://github.com/BindsNET/bindsnet/issues and ask if they tried to get transferred the pypi namespace?

@conda-forge/staged-recipes any input?

Reply:
This is a really hard questions. We usually treat the PyPI package metadata as the canonical source of information but, they package there may be outdated and we don't know the story behind the two repos.

@Tobias-Fischer do you mind asking upstream which one is the canonical source and/or the story behind the fork? Are the same people involved in the active project? Are the packages compatible?

Reply:
Sure - let's see: https://github.com/BindsNET/bindsnet/issues/560

Reply:
Thanks! Keep us posted here. Hopefully they'll get back to you quickly.

Reply:
Hiya - I heard back from the team at https://github.com/BindsNET/bindsnet/ via email. It sounds like a typical story around two forks that have split over time. The person who has the PyPi keys has left academia without passing on the keys, so there is little hope of new updates on PyPi. Considering that https://github.com/BindsNET/bindsnet/ clearly has more recent updates and is much more active, I'd strongly argue in favour of releasing their versions here on conda-forge. It would be a shame not having it on conda-forge for this "technicality". Any thoughts @beenje @ocefpaf?

Reply:
> The person who has the PyPi keys has left academia without passing on the keys, so there is little hope of new updates on PyPi.

:disappointed: not the first time, probably not the last. Let's merge this. Thanks @Tobias-Fischer and @beenje for the clarifications!